io.github.rsvzz.csven/io.github.rsvzz.csven.json

```
@@ -0,0 +1,60 @@
+{
+  "id": "io.github.rsvzz.csven",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "47",
+  "base-version": "0.0.3",
+  "sdk": "org.gnome.Sdk",
+  "command": "csven",
+  "branch": "stable",
+  "finish-args": [
+      "--socket=session-bus",
+      "--share=ipc",
+      "--socket=wayland",
+      "--socket=x11",
+      "--socket=fallback-x11",
+      "--device=dri",
+      "--env=DISPLAY=:0",
+      "--env=GTK_THEME=Adwaita:dark",
+      "--env=LD_LIBRARY_PATH=/app/lib"
+  ],
+  "modules": [
+    {
+      "name": "questk",
+      "buildsystem": "meson",
+      "config-opts": [
+        "--prefix=/app",
+        "--buildtype=release",
+        "--libdir=lib"
```

```suggestion
        "--buildtype=release"
```

this is set properly by default

---

ready